services/gateway/tests/constants/utils.js

```diff
@@ -0,0 +1,267 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const transformParamsInput = {
+	'key to be overwritten': {
+		altSwaggerKey: 'custom swagger key',
+	},
+	sort: {
+		type: 'sort type',
+		enum: 'sort enum',
```

Comment:
All the `enum` values must be a list of `type`. Let's keep the test fixtures realistic. Applies to all the entries in the file.

Eg: For `sort` param, the `type` would be string and `enum` would be a list of strings. Please adjust accordingly.